Ruby.gitignore

```diff
@@ -9,6 +9,7 @@
 /test/tmp/
 /test/version_tmp/
 /tmp/
+tags
```

Comment:
I'm not sure what this file represents, and why it can be ignored in version control? Could you elaborate?